automated install/basic-install.sh

```diff
@@ -1015,10 +1015,12 @@ valid_ip() {
     local ip=${1}
     local stat=1
 
-    # One IPv4 element is 8bit: 0 - 256
-    local ipv4elem="(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)";
-    # optional port number starting '#' with range of 1-65536
-    local portelem="(#([1-9]|[1-8][0-9]|9[0-9]|[1-8][0-9]{2}|9[0-8][0-9]|99[0-9]|[1-8][0-9]{3}|9[0-8][0-9]{2}|99[0-8][0-9]|999[0-9]|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-6]))?"
+    # One IPv4 element is 8bit: 0 - 255
+    # See https://tools.ietf.org/html/rfc1340
+    local ipv4elem="(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[0-9]{1,2})";
+    # optional port number starting ':' with range of 0-65535
+    # See https://tools.ietf.org/html/rfc6346
```

Comment:
Which section, I guess you mean the introduction of RFC 6346? Anyway, [RFC 6335, Section 6](https://tools.ietf.org/html/rfc6335#section-6) would be a better reference here.

Reply:
Or maybe even [RFC 1340 (page 33)](https://tools.ietf.org/html/rfc1340#page-33)

Reply:
I'd prefer to reference RFC1340 (since it's not listed as experimental). Thanks for the links.